service/src/builder.rs

```diff
@@ -75,7 +130,15 @@ pub async fn build_workspace(
         let mut alpha_binaries = compilation
             .binaries
             .iter()
-            .map(|binary| Runtime::Alpha(binary.path.clone()))
+            .map(|binary| {
+                BuiltService::new(
+                    binary.path.clone(),
+                    false,
+                    binary.unit.pkg.name().to_string(),
+                    binary.unit.pkg.root().to_path_buf(),
+                    binary.unit.pkg.manifest_path().to_path_buf(),
```

Comment:
Maybe we should store the entire `binary` or `binary.unit` and thereby have access to anything in the future?

Reply:
Sure! I didn't do it because I was trying to be less dependent on `cargo` since we're planning to remove it eventually, but that will require a bit of work here anyway to get all the required data.

Reply:
Ah, Unit is also !Send and !Sync :thinking:

Reply:
We'd have to put in a mutex or something, so I may just stick with the current implementation now and see if I can find a smarter way when working on deployer.